### PR TITLE
21 main view branch section

### LINKED ISSUE #21: Main view Branch section
# REQUIREMENTS
Create view to showw list of branches following the next design

# DESIGN
imagen

### PR DESCRIPTION
**Pull request by femm34**
# Description
What did you implemented
 - added a new method to search branches by name 
 - added new styles to branches main view
 
Why did you implemented this:
 -  search method was added to make it faster to search for any branch

  ## Screenshots
image
image
image

### CODE REVIEW

File: app/javascript/controllers/branches_controller.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { Controller } from "@hotwired/stimulus";
+
+// Connects to data-controller="branches"
+export default class extends Controller {
+  static targets = ["form_branches", "button_new"];
+  // static targets = ["button_new_branch"]
+
+  connect() {
+    console.log("Hello, Stimulus!", this.element);
```
Comment (YilKanda) [inline app/javascript/controllers/branches_controller.js]: It is not recommended to leave debugging code like console.log()

File: app/javascript/controllers/branches_controller.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { Controller } from "@hotwired/stimulus";
+
+// Connects to data-controller="branches"
+export default class extends Controller {
+  static targets = ["form_branches", "button_new"];
+  // static targets = ["button_new_branch"]
```
Comment (YilKanda) [inline app/javascript/controllers/branches_controller.js]: Code that you do not occupy delete it.

**Review by YilKanda — CHANGES_REQUESTED** [review]

File: app/javascript/controllers/branches_controller.js (hunk 1 above)
Comment (femm34) [reply app/javascript/controllers/branches_controller.js]: @YilKanda thanks, im gonna delete it

File: app/javascript/controllers/branches_controller.js (hunk 2 above)
Comment (femm34) [reply app/javascript/controllers/branches_controller.js]: @YilKanda thansk , im gonna do it

**Review by YilKanda — APPROVED** [review]

**Review by EliasUR — APPROVED** [review]
LGTM ;)

**Review by AlejandroCuriel — APPROVED** [review]
LGTM :rocket: :eagle:

**Pull request merged**